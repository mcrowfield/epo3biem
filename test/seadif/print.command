echo "Layout to Postscript printer command sequence" ; echo running getepslay -t Tmppos11997 ; getepslay -t Tmppos11997 ; echo printing Tmppos11997.eps ; lpr Tmppos11997.eps ; lpq ; rm -f Tmppos11997.eps ; echo terminate >> seadif/print.out

