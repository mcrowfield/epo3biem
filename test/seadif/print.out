Layout to Postscript printer command sequence
running getepslay -t Tmppos11997
makeboxl: warning: mask_copies not more supported!
makegln: Temp file compression turned off
printing Tmppos11997.eps
seadif/print.command: line 1: lpr: command not found
seadif/print.command: line 1: lpq: command not found
terminate
terminate
